MANIFEST.in

```diff
@@ -1,5 +1,7 @@
 include *.txt
 include *.rst
+exclude buildout.cfg
+exclude tox.ini
```

Comment:
I like things like `buildout.cfg` and `tox.ini` in my sdists.  They provide useful information pertaining to the source (here's how you can build a dev env, here's how you can run tests).

I suppose nobody sane starts working on code by unpacking a tarball nowadays, it's always a git checkout, so feel free to disregard my comment.